hadoop-hdds/rocks-native/src/main/java/org/apache/hadoop/hdds/utils/db/managed/PipeInputStream.java

```diff
@@ -43,7 +39,8 @@ public class PipeInputStream extends InputStream {
   private AtomicBoolean cleanup;
 
   PipeInputStream(int capacity) throws NativeLibraryNotLoadedException {
-    if (!NativeLibraryLoader.isLibraryLoaded(ROCKS_TOOLS_NATIVE_LIBRARY_NAME)) {
+    if (!NativeLibraryLoader.getInstance()
```

Comment:
qq: do we need to load lib again here or check? Isn't it not enough loading it as part of ManagedSSTDumpTool instantiation?

Reply:
we need to check if the library is loaded, instantiation of the class should fail

Reply:
done